src/sparsezoo/analyze/utils/models.py

```diff
@@ -374,7 +374,16 @@ class Section(Entry):
     Represents a list of Entries with an optional name
     """
 
-    entries: List[Union[NamedEntry, TypedEntry, SizedModelEntry, ModelEntry, BaseEntry]]
+    entries: List[
```

Comment:
different formatter?

Reply:
Nope, confirmed this is done by `make style && make quality` and not my IDE